### ISSUE TITLE
Providing specific `return_fields` in `get_parameter_list` doesn't work

### ISSUE THREAD

**Issue by callum-savage** [callum-savage/bomWater#7]
When specified, only the default parameters are returned. 

I need to double check this.

**Comment by callum-savage** [comment]
Reprex: return fields is being completely ignored in favour of the defaults

``` r
bomWater::get_parameter_list(station_number = "422205B", return_fields = c("station_number", "parametertype_name"))
#> # A tibble: 2 × 6
#>   station_name station_no station_id parametertype_id parametertype_name    
#>   <chr>        <chr>           <int>            <int> <chr>                 
#> 1 Hastings     422205B      19951521            11762 Water Course Discharge
#> 2 Hastings     422205B      19951521            11763 Water Course Level    
#> # ℹ 1 more variable: stationparameter_name <chr>
```

<sup>Created on 2023-05-23 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by callum-savage** [comment]
Closed by f15629e8
